test/CMakeLists.txt

```diff
@@ -36,8 +36,8 @@ find_package(Boost 1.67.0 REQUIRED COMPONENTS unit_test_framework)
 if(NOT DEFINED Boost_VERSION_MINOR)
   set(Boost_VERSION_MINOR ${Boost_MINOR_VERSION})
 endif()
-if(Kokkos_ENABLE_CUDA AND Boost_VERSION_MINOR GREATER_EQUAL 69)
-  message(FATAL_ERROR "Tests in CUDA-enabled ArborX are only compatible with Boost version 1.67 or 1.68")
+if(Kokkos_ENABLE_CUDA AND Boost_VERSION VERSION_GREATER 1.68 AND Boost_VERSION VERSION_LESS 1.75)
```

Comment:
Note the change from `Boost_VERSION_MINOR` to `Boost_VERSION`. I *think* it should be OK, but could you please confirm? There was some change in its meaning in 3.15 and I'm also not sure Boost versions prior to 1.70 provide it correctly.

Reply:
I tested locally.